src/Http/MiddlewareQueue.php

```diff
@@ -145,7 +145,7 @@ public function prepend($middleware)
      */
     public function insertAt($index, $middleware)
     {
-        array_splice($this->queue, $index, 0, $middleware);
+        array_splice($this->queue, $index, 0, [$middleware]);
```

Comment:
I was wondering if it is necessary to first check if `$middleware` is an array but I didn't find any cases.
Let me know if you feel it's important.